src/lxml/html/diff.py

```diff
@@ -473,7 +467,7 @@ def __new__(cls, text, pre_tags=None, post_tags=None, trailing_whitespace=""):
         return obj
 
     def __repr__(self):
-        return 'token(%s, %r, %r, %r)' % (_unicode.__repr__(self), self.pre_tags,
+        return 'token({}, {!r}, {!r}, {!r})'.format(_unicode.__repr__(self), self.pre_tags,
                                           self.post_tags, self.trailing_whitespace)
```

Comment:
Interesting. Why not an f-string here?

Reply:
I think it didn't recognize it, I could manually convert it.